adcm_aio_client/core/objects/cm.py

```diff
@@ -41,6 +41,26 @@ def get_own_path(self: Self) -> Endpoint:
         return ("adcm",)
 
 
+class License(WithRequester):
+    def __init__(self: Self, bundle_data: dict) -> None:
+        self._license_prototype_id = bundle_data["mainPrototype"]["id"]
+        self._data = bundle_data["license"]
+
+    @property
+    def text(self: Self) -> str:
+        return str(self._data["license"]["text"])
+
+    @property
+    def state(self: Self) -> Literal["absent", "accepted", "unaccepted"]:
+        return self._data["license"]["status"]
+
+    async def accept(self: Self) -> str:
+        response = await self._requester.post("prototypes", self._license_prototype_id, "license", "accept", data={})
```

Comment:
requester can't be accessed from here

`WithRequester` is just a protocol

I suggest `"prototypes", self._license_prototype_id, "license", "accept"` to be passed in constructor

Because `bundle_data` binds license to `Bundle` too much

So we can have something like `self._data = data` and go from it

current approach already lead to confusion:

```python
    @property
    def text(self: Self) -> str:
        return str(self._data["license"]["text"])

    @property
    def state(self: Self) -> Literal["absent", "accepted", "unaccepted"]:
        return self._data["license"]["status"]
```

double license retrieval, etc.